proposals/spi-proposal.md

```diff
@@ -112,168 +105,150 @@ MSBFIRST = 1
 | 2       | 1                     | 0                  |
 | 3       | 1                     | 1                  |
 
-
 #### maxSpeed
 
-The maximum speed of communication with the SPI device. For a SPI device rated up to
-5 MHz, use 5000000.
-
-#### wordSize
+The maximum speed of communication with the SPI device. For a SPI device rated
+up to 5 MHz, use 5000000.
 
-The size of a `word` in bits. Typically 8-bits (default). 0 = DEFAULT = 8-bits, 1 = 1 bit,
-2 = 2 bits, etc (limit is TBD).
+*For platforms that use a clock divider instead of a speed, pass the clock
+divider value instead.* 
 
-#### csPin / csPinOptions
+#### wordSize
 
-Specify a CS (chip select) pin to automatically toggle the pin rather than having to
-send a separate `DIGITAL_MESSAGE` command with every transfer.
+The size of a `word` in bits. Typically 8-bits (default). 0 = DEFAULT = 8-bits,
+1 = 1 bit, 2 = 2 bits, etc (limit is TBD).
 
-The default behavior is to pull the CS pin LOW at the beginning of a transfer and toggle
-it at the end of the transfer. Setting bit 0 (`CS_ACTIVE_STATE`) will pull the CS pin HIGH
-at the start of the transfer. Set bit 1 (`CS_TOGGLE`) to handle the rare use case of
-toggling the CS pin between every word sent rather than at the end of the transaction.
+#### csPinOptions / csPin
 
-The CS pin behavior can be further controlled by using the `csPinControl` byte of the
-`SPI_TRANSFER`, `SPI_WRITE` or `SPI_READ` messages.
+Use `csPinOptions` to set the active state (typically LOW) for the CS pin. If
```

Comment:
@fivdi Please let me know if this description makes more sense now.

Reply:
Yes, the description make sense.

Maybe it would be a little clearer if the first sentence was changed from:

Use `csPinOptions` to set the active state (typically LOW) for the CS pin.

to:

Use `CS_ACTIVE_STATE` to set the active state (typically LOW) for the CS pin.

Reply:
Sounds good.